ipn/ipnlocal/c2n.go

```diff
@@ -77,3 +88,92 @@ func (b *LocalBackend) handleC2N(w http.ResponseWriter, r *http.Request) {
 		http.Error(w, "unknown c2n path", http.StatusBadRequest)
 	}
 }
+
+func (b *LocalBackend) handleC2NUpdate(w http.ResponseWriter, r *http.Request) {
```

Comment:
Thought for the future: we may want some sort of semaphore that prevents two concurrent updates, or if one happened in the past 5 minutes, or something.

Reply:
Added TODO with that text